specification/frontdoor/resource-manager/Microsoft.Network/preview/2018-08-01-preview/frontdoor.json

```diff
@@ -1808,6 +1821,67 @@
             "name": "FrontDoorEnabledState",
             "modelAsString": true
           }
+        },
+        "redirectConfiguration": {
+          "description": "A reference to the redirect routing configuration. ",
+          "$ref": "./network.json#/definitions/RedirectConfiguration"
+        }
+      }
+    },
+    "RedirectConfiguration": {
+      "description": "The configuration for a redirect routing rule. This object is needed only if the type property of RoutingRule is set to Redirect. ",
+      "type": "object",
+      "properties": {
+        "redirectType": {
+          "description": "The redirect type the rule will use when redirecting traffic.",
+          "type": "string",
+          "enum": [
+            "Moved(301)",
+            "Found(302)",
+            "TemporaryRedirect(307)",
+            "PermanentRedirect(308)"
+          ],
+          "x-ms-enum": {
+            "name": "FrontDoorRedirectProtocol",
+            "modelAsString": true
+          }
+        },
+        "destinationProtocol": {
+          "description": "The protocol of the destination where the traffic is forwarded to",
+          "type": "string",
+          "enum": [
+            "MatchRequest",
+            "Htt",
```

Comment:
Is this `Http`?

Reply:
Yes, it should be Http. Thanks for catching it.